testing/test_browser.py

```diff
@@ -78,6 +78,13 @@ def test_wait_for_element_visible_fail_except(browser):
         browser.wait_for_element('#invisible_appear_p', visible=True, timeout=1.5)
 
 
+def test_wait_for_element_visible_fail_none(browser):
+    # Click on the button
+    browser.click('#invisible_appear_button')
+    assert browser.wait_for_element(
+        '#invisible_appear_p', visible=True, timeout=1.5, exception=False) is None
```

Comment:
Can you please expand/parametrize the test to also pass exception=True in a `pytest.raises` block?

Reply:
nvm, I see its separate test case already.

Reply:
I can, but isn't this scenario already covered in `test_wait_for_element_visible_fail_except`? Or you want to check explicitly?

Reply:
It is, I'd like to combine them, do you mind if I push a commit to your PR with an updated test?  I'm targeting to merge and release this now(ish)

Reply:
sure thing

Reply:
Making sure updated tests pass in travis, they did locally in Tox.